ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/AtomicActionTestBase.java

```diff
@@ -50,7 +50,7 @@ protected void testCommit () throws Exception
 
     protected void testAbort () throws Exception
     {
-        executeTest(true, ActionStatus.COMMITTED, null, new BasicRecord(), new BasicRecord());
+        executeTest(false, ActionStatus.ABORTED, null, new BasicRecord(), new BasicRecord());
```

Comment:
Good catch. Mind, you also need to set the second parameter (expectedResult) to ActionStatus.ABORTED.